client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -81,6 +81,17 @@ export const prepareExtensionTrackingData = (
 	return installedExtensions;
 };
 
+export const isSellingElsewhere = ( selectedOption ) =>
+	[
+		'other',
+		'brick-mortar',
+		'brick-mortar-other',
+		'other-woocommerce',
+	].includes( selectedOption );
+
+export const isSellingOtherPlatformInPerson = ( selectedOption ) =>
```

Comment:
Would it still make sense if we shorten this down to `isSellingInPerson`?

Reply:
@joshuatf I like the const name you suggested but I'm hesitating because `isSellingInPerson` could be a little confusing since it seems like it's referring to the option `Yes, in person at physical stores and/or events` instead of `Yes, on another platform and in person at physical stores and/or events`.
What do you think about: `isSellingOtherPlatformIP` or `isAnotherPlatformAndIP`?

Reply:
Actually, I'd prefer the original name `isSellingOtherPlatformInPerson` better since it's more accurate and easy to pick up by reading quickly. I don't think that's too long to be a problem IMO.

Reply:
That's a good point, @octaedro.  Let's keep the variable name as is for now 👍